### ISSUE TITLE
Possible fix needed: NITs and SO4s should be wet removed as coarse aerosol species. 

### ISSUE THREAD

**Issue by msl3v** [geoschem/geos-chem#1414]
Since wetscav_mod.F90 distinguishes between COARSE and FINE aerosol in calculating washout rates, it needs to be pointed out that neither NITs nor SO4s are washed out as coarse species. It seems this is changed simply by setting  WD_CoarseAer: true
in species_database.yml for these two species. Testing in the GEOS-CF system shows this has a significant impact on NITs and SO4s.

**Comment by yantosca** [comment]
Thanks for raising the issue @msl3v.  Am tagging the Aerosols WG co-chairs: @theloniuspunk @beckyalexander @wporter

**Comment by yantosca** [comment]
Also tagging @Jourdan-He and @SaptSinha

**Comment by beckyalexander** [comment]
SO4s and NITs are course-mode aerosols and should be treated exactly like coarse-mode sea salt aerosols, including using the same molecular weight, etc as the coarse-mode sea salt aerosols. SO4s and NITs are essentially internally mixed with coarse-model sea salt aerosols.

**Comment by yantosca** [comment]
Thanks @msl3v and @beckyalexander. This could be a fix for 14.1.0, as it would only involve changing the species database metadata for those species.  I'll tag it as such.

**Comment by yantosca** [comment]
Follow-up question:  In the `species_database.yml` file, we have 
```yaml
SALC_PROP: &SALCproperties
  DD_AeroDryDep: true
  DD_F0: 0.0
  DD_Hstar: 0.0
  Density: 2200.0
  Is_Advected: true
  Is_Aerosol: true
  Is_DryDep: true
  Is_WetDep: true
  Radius: 4.25e-6
  WD_AerScavEff: 1.0
  WD_KcScaleFac: [1.0, 0.5, 1.0]
  WD_RainoutEff: [1.0, 0.0, 1.0]
  WD_RainoutEff_Luo: [0.4, 0.0, 1.0]
...
NITs:
  << : *SALCproperties
  FullName: Inorganic nitrates on surface of seasalt aerosol
  Is_Photolysis: true
  MW_g: 31.4
...
SO4s:
  << : *SALCproperties
  FullName: Sulfate on surface of seasalt aerosol
  MW_g: 31.4
```
So in other words, the metadata listed under `SALC_PROPERTIES` are common to several species (BrSALC, ISALC, NITs, SALC, SALCAL, SALCCL, SO4s),

Should we apply `WD_CoarseAer:True` to:

  1. Just for SO4s and NITs, or
  2. To `SALC_PROPERTIES` (and thus it would carry over to all of the species listed above)

Because BrSALC is one of the species, do you have a sense if this would also significantly change the Bromine chemistry?

Also tagging @djxjacob for advice.

**Comment by msl3v** [comment]
Hey Bob, 
BrSALC already has "WD_CoarseAer: true" as an attribute, in addition to *SALCproperties, so it would not have an effect to add it to SALCproperties.

**Comment by yantosca** [comment]
Thanks Mike (@msl3v).  Will proceed.

**Comment by yantosca** [comment]
We can close this issue as PR #1426 has been merged into the GEOS-Chem 14.1.0 development stream.
